src/gametest/groovy/techreborn/test/machine/GrinderTest.groovy

```diff
@@ -32,7 +32,7 @@ import techreborn.test.TRGameTest
 import techreborn.test.TRTestContext
 
 class GrinderTest extends TRGameTest {
-    @GameTest(structureName = EMPTY_STRUCTURE, tickLimit = 150)
+    @GameTest(structureName = fabric-gametest-api-v1:empty, tickLimit = 150)
```

Comment:
I dont believe this is needed as it was caused by a bug in the version of groovy the 1.18 branch is using.

Reply:
couldn't get it to build without adding this, failed immediately even with just the 1.17 master, no changes.

Reply:
Humm odd, leave it then. I didn’t bother to look into it any more when porting to 1.18

Reply:
build failed on git, but been playing it all evening without any issues.